apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V72__new_schema_inpatient_claims.sql

```diff
@@ -0,0 +1,960 @@
+-- NEW_SCHEMA_INPATIENT_CLAIMS.SQL
+-- flyway migration for INPATIENT_CLAIMS and INPATIENT_CLAIM_LINES tables into
+-- a new schema structure that:
+--   1) changes data type of CLM_ID, BENE_ID from varchar to BIGINT.
+--   2) change data type of CLM_GROUP_ID from numeric to BIGINT
+--   3) organizes parent claim table (INPATIENT_CLAIMS) such that common
+--      claims data columns are organized at top of table structure.
+--
+--      The following db columns were redefined from NUMERIC to more
+--      appropriate data type(s):
+--
+--           line_num - changed to smallint
```

Comment:
*clm_line_num

Should this list also include CLM_GROUP_ID?

Reply:
clm_group_id is called out on line 5. thx for catcin the line_num....it gets confusing since some of the tables have clm_line_num and others have line_num. I had fixed all that in the original ticket (BFD-1010) but it was deemed too unwieldy to review, so we broke things up and some of the normalization got axed for the time being.